File: cpp/bc1042.cpp
#include <iostream>

using namespace std;

int main()
{
	int fVal, sVal, tVal;
	
	cin >> fVal >> sVal >> tVal;
	
	int cOrg[3] = {fVal, sVal, tVal};
	
	for(int i = 0; i < 3; i++)
	{
		for(int j = 0; j < 2; j++)
		{
			if(cOrg[j] > cOrg[j+1])
			{
				int aux = cOrg[j];
				cOrg[j] = cOrg[j+1];
				cOrg[j+1] = aux;
			}
		}
	}
	
	for(int k = 0; k < 3; k++) cout << cOrg[k] << endl;
	
	cout << "\n" << fVal << "\n" << sVal << "\n" << tVal << endl;
}

File: python/bc1042.py
cVal = input().split(" ")

fVal = int(cVal[0])
sVal = int(cVal[1])
tVal = int(cVal[2])

for j in range(0,3):
    for i in range(0,2):
        if int(cVal[i]) > int(cVal[i+1]):
            aux = cVal[i+1]
            cVal[i+1] = cVal[i]
            cVal[i] = aux;

for c in range(0,3): print(cVal[c])

print(f"\n{fVal}\n{sVal}\n{tVal}")